eo-parser/src/main/java/org/eolang/parser/DrErrors.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2024 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.parser;
+
+import java.util.Iterator;
+import org.cactoos.iterable.Mapped;
+import org.xembly.Directive;
+import org.xembly.Directives;
+
+/**
+ * Error directives.
+ * @since 0.50
+ */
+final class DrErrors implements Iterable<Directive> {
+
+    /**
+     * Errors accumulated.
+     */
+    private final Errors errors;
+
+    /**
+     * Ctor.
+     * @param errors The errors.
+     */
+    DrErrors(final Errors errors) {
+        this.errors = errors;
```

Comment:
@volodya-lombrozo here, you only use `errors.all()` method here, don't use `size()`. Why can't you use `Iterable` instead of `Errors` here?